release/ray_release/buildkite/step.py

```diff
@@ -94,15 +85,8 @@ def get_step(
     env_dict.update(env)
 
     step["env"].update(env_dict)
-
-    if "python" in test:
-        python_version = parse_python_version(test["python"])
-    else:
-        python_version = DEFAULT_PYTHON_VERSION
-
-    step["plugins"][0][DOCKER_PLUGIN_KEY][
-        "image"
-    ] = f"rayproject/ray:nightly-py{python_version_str(python_version)}"
+    step["plugins"][0][DOCKER_PLUGIN_KEY]["image"] = "python:3.8"
```

Comment:
you can also use a digest (like `python@sha256:xxxxx`) to make it more deterministic.

Reply:
I'll keep it as python:3.8 and trust their upgrades will not cause too much trouble for now ;)